composer.json

```diff
@@ -12,14 +12,12 @@
     "require": {
         "php": "^7.2",
         "symfony-cmf/resource-bundle": "^1.0",
-        "jms/serializer-bundle": "^1.0 | ^2.0 | ^3.0",
-        "symfony/translation": "^2.8 || ^3.3 || ^4.0"
+        "jms/serializer-bundle": "^2.0 | ^3.0"
     },
     "require-dev": {
         "symfony-cmf/testing": "^2.1@dev",
         "symfony/phpunit-bridge": "^5",
         "doctrine/phpcr-odm": "^1.4|^2.0",
-        "jms/serializer": "^1.2",
```

Comment:
i think this is the reason why the lowest-version test now fails. which means we should actually put this into the conflict section below, as `"jms/serializer": "<1.2.0"`

Reply:
But it's already included in the package https://github.com/schmittjoh/JMSSerializerBundle/blob/3.7.0/composer.json#L26

Reply:
i agree that we should not have it here as a dev-dependency.

but look at the failing test on travis-ci - they indicate that the visitor interface does not exist. which means that we ended up with a too old version of this package. but should ensure that nobody installs this bundle with an incompatible version, hence we should mark the conflict.

Reply:
@dbu found out, that only serializer-bundle 3.0 uses jms serializer 2.0. I dropped jms/serializer-bundle 2.0 in a fix. Waiting for travis results...